zkevm-circuits/src/keccak_circuit/keccak_bit.rs

```diff
@@ -76,6 +79,11 @@ pub(crate) struct KeccakRow {
     c_bits: [u8; C_WIDTH],
     a_bits: [u8; KECCAK_WIDTH / 25],
     q_end: u64,
+
+    // padding
+    // s_flags: [bool; KECCAK_RATE / 8],
+    d_bits: [u8; KECCAK_RATE],
```

Comment:
We need to reduce 1088 columns into 44 columns as we have 25 rows. The current implementation in PR has 25 rows of duplicated data. Each row has the same 1088 columns.